functions.opengraph.php

```diff
@@ -261,22 +261,20 @@ function jetpack_og_tags() {
 }
 
 function jetpack_og_get_image( $width = 200, $height = 200, $max_images = 4 ) { // Facebook requires thumbnails to be a minimum of 200x200
-	$image = '';
+	$image = array();
 
 	if ( is_singular() && ! is_home() ) {
 		global $post;
```

Comment:
Normally I'd leave it, since it's only used within that conditional, but now that you've called it out, I'm inclined to remove it entirely. It's only there to provide access to the post ID, which `get_the_ID()` can do instead, simplifying this section. Update forthcoming.